gitpod-io/website
# Quotes on `/vs/github-codespaces` page

## Description

Quotes on `vs/codespaces` page

## Related Issue(s)



Fixes #3510

## How to test



- Open [`/vs/github-codespaces`](https://deploy-preview-3512--gitpod-kumquat.netlify.app/vs/github-codespaces) & see :eyes:

## Release Notes



```release-note
NONE
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3512"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
